windows-observ-lib/targets.libsonnet

```
@@ -335,5 +334,84 @@ local lokiQuery = g.query.loki;
         'irate(windows_net_packets_sent_total{%(queriesSelector)s}[$__rate_interval])' % variables
       )
       + prometheusQuery.withLegendFormat('{{ nic }} transmitted'),
+
+    replicationPendingOperations:
+      prometheusQuery.new(
+        '${' + variables.datasources.prometheus.name + '}',
+        'sum(windows_ad_replication_pending_operations{%(queriesSelector)s})' % variables
+      )
+      + prometheusQuery.withLegendFormat('Operations'),
+    directoryServiceThreads:
+      prometheusQuery.new(
+        '${' + variables.datasources.prometheus.name + '}',
+        'sum(windows_ad_directory_service_threads{%(queriesSelector)s})' % variables
+      )
+      + prometheusQuery.withLegendFormat('Directory service threads'),
+    replicationPendingSynchronizations:
+      prometheusQuery.new(
+        '${' + variables.datasources.prometheus.name + '}',
+        'sum(windows_ad_replication_pending_synchronizations{%(queriesSelector)s})' % variables
+      )
+      + prometheusQuery.withLegendFormat('Operations'),
+    ldapBindRequests:
+      prometheusQuery.new(
+        '${' + variables.datasources.prometheus.name + '}',
+        'rate(windows_ad_binds_total{bind_method=~"ldap", %(queriesSelector)s}[$__rate_interval])' % variables
+      )
+      + prometheusQuery.withLegendFormat('{{ instance }}'),
```

Perhaps we can be more generic here, so legend is transformed if something else, other than instance is used:

```suggestion
      + prometheusQuery.withLegendFormat(commonlib.utils.labelsToPanelLegend(this.config.instanceLabels)),
```

Please also add import to the top of the file:
`local commonlib = import 'common-lib/common/main.libsonnet';`